plugins/content-highlight/README.md

```diff
@@ -0,0 +1,40 @@
+# workspace-content-highlight [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
```

Comment:
(Can be a followup) Please add an image to show what this one does--it's much clearer than a written description.

Reply:
Done.